dc/utils.py

```diff
@@ -0,0 +1,30 @@
+import six.moves.urllib
+from Bio import Entrez, Medline
+import re
+
+Entrez.email = "trivneel211@gmail.com"
+
+
+def get_pubmed_ids_from_phenotypes(phenotypes):
+    pubmed_ids = []
+    # TODO
```

Comment:
didn't we finish this already?  @NeelayTrivedi

Reply:
I think we have a prototype in https://github.com/NCBI-Hackathons/DiseaseClusters/blob/a53b748e79a953334d88c6eab1771a59011d4d3d/get_abstracts.py#L87-L98

Reply:
However, I think the conclusion we have yesterday is to use PubMed search directly.